test/ExecCommand.js

```diff
@@ -25,6 +25,50 @@ describe("ExecCommand", () => {
     }));
   });
 
+  it("should filter packages with `ignore`", (done) => {
+    const execCommand = new ExecCommand(["ls"], {});
+
+    execCommand.runValidations();
+    execCommand.runPreparations();
+
+    assert.equal(execCommand.packages.length, 2);
+    assert.equal(execCommand.packages[0].name, "package-1");
+    assert.equal(execCommand.packages[1].name, "package-2");
+
+    execCommand.flags.ignore = "package-1";
```

Comment:
lines 34-38 are not necessary if you pass the `ignore` flag as part of the constructor.